redisgraph/edge.py

```diff
@@ -14,9 +14,9 @@ def __init__(self, src_node, relation, dest_node, edge_id=None, properties=None)
 
         self.id = edge_id
         self.relation = '' or relation
```

Comment:
@swilly22 don't you need for relation too?

Reply:
https://github.com/RedisGraph/redisgraph-py/blob/master/redisgraph/edge.py#L19 it actually already fixed